test/Index/index_module_refs.swift

```diff
@@ -0,0 +1,30 @@
+// RUN: %target-swift-ide-test -print-indexed-symbols -enable-source-import -source-filename %s -I %S/Store/Inputs | %FileCheck %s
+
+import ClangModuleB
+// CHECK: [[@LINE-1]]:8 | module/C | ClangModuleB | c:@M@ClangModuleB | Ref | rel: 0
+import ClangModuleC.Sub1
+// CHECK: [[@LINE-1]]:8 | module/C | ClangModuleC | c:@M@ClangModuleC | Ref | rel: 0
+// CHECK: [[@LINE-2]]:21 | module/C | ClangModuleC.Sub1 | c:@M@ClangModuleC@M@Sub1 | Ref | rel: 0
+import ClangModuleC.Sub2
+// CHECK: [[@LINE-1]]:8 | module/C | ClangModuleC | c:@M@ClangModuleC | Ref | rel: 0
+// CHECK: [[@LINE-2]]:21 | module/C | ClangModuleC.Sub2 | c:@M@ClangModuleC@M@Sub2 | Ref | rel: 0
+import func ClangModuleA.funcA
+// CHECK: [[@LINE-1]]:13 | module/C | ClangModuleA | c:@M@ClangModuleA | Ref | rel: 0
+// CHECK: [[@LINE-2]]:26 | function/Swift | funcA() | c:@F@funcA | Ref | rel: 0
+import SwiftModuleC
+// CHECK: [[@LINE-1]]:8 | module/Swift | SwiftModuleC | c:@M@SwiftModuleC | Ref | rel: 0
+
+struct MyType {}
+
+extension SwiftModuleC.MyType {
+  // CHECK: [[@LINE-1]]:11 | module/Swift | SwiftModuleC | c:@M@SwiftModuleC | Ref | rel: 0
+  func myMethod() {
+    _ = SwiftModuleC.MyGeneric<SwiftModuleC.MyType, MyType>()
+    // CHECK: [[@LINE-1]]:9 | module/Swift | SwiftModuleC | c:@M@SwiftModuleC | Ref | rel: 0
+    // CHECK: [[@LINE-2]]:22 | struct/Swift | MyGeneric | s:12SwiftModuleC9MyGenericV | Ref,RelCont | rel: 1
```

Comment:
I think it will be a bit better to keep the test case focused on testing that module references are indexed. Meaning we don't need to check for `MyGeneric` and `MyType` here, other test cases should be responsible to make sure that we index these properly.
Essentially I'd recommend to remove the check lines for those.

Reply:
That's fair, it'd be fine to have the additional checks but at least reduce how strict the CHECK string needs to be, for example if we only want to check that the `struct` reference shows up we could have it like this:

```
// CHECK: [[@LINE-2]]:22 | struct/Swift | MyGeneric | {{.*}} | Ref
```

So essentially we care that `MyGeneric` reference shows up but we don't care to match its USR.